src/app/+display/tab/tab.component.html

```diff
@@ -1,9 +1,7 @@
-<div *ngIf="individual | async; let individual" class="tab-content">
-  <div *ngIf="tab | async; let tab" class="tab-pane active">
-    <div *ngIf="getSectionsToShow(tab, individual) as sections ; else no_content">
-      <div class="ml-3 mr-3 mt-3 card" *ngFor="let section of sections">
-        <scholars-section [section]="section" [display]="display" [individual]="individual"></scholars-section>
-      </div>
+<div *ngIf="individual | async; let individual" class="tab-content" role="tabpanel" aria-live="polite">
+  <div *ngIf="tab | async; let tab" class="tab-pane active" [attr.aria-labelledby]="tab.id + '-tab'" role="tabpanel">
+    <div class="ml-3 mr-3 mt-3 card" *ngFor="let section of getSectionsToShow(tab, individual)">
+      <scholars-section [section]="section" [display]="display" [individual]="individual" aria-labelledby="section.id + '-section'"></scholars-section>
```

Comment:
This one makes more sense.
You are using teh `secion.id` and concatting the `-section`.
This should be more distinct.
The other generic named `id` tags out there for generic components have me concerned.

Having said that, I am getting a lot of:
> Broken ARIA reference
An aria-labelledby or aria-describedby reference exists, but the target for the reference does not exist.

Involving:
```html
<scholars-section _ngcontent-ng-c1691634590="" aria-labelledby="section.id + '-section'" _nghost-ng-c2676255389="" ng-reflect-section="[object Object]" ng-reflect-display="Documents" ng-reflect-individual="[object Object]">
```